doc/connectivity/bluetooth/api/audio/shell/cap.rst

```diff
@@ -257,15 +257,19 @@ the optionally included CSIS instance by calling (:code:`cap_commander discover`
    cap_commander --help
    cap_commander - Bluetooth CAP commander shell commands
    Subcommands:
-     discover                :Discover CAS
-     cancel                  :CAP commander cancel current procedure
-     change_volume           :Change volume on all connections <volume>
-     change_volume_mute      :Change volume mute state on all connections <mute>
-     change_volume_offset    :Change volume offset per connection <volume_offset
-                              [volume_offset [...]]>
-     change_microphone_mute  :Change microphone mute state on all connections <mute>
-     change_microphone_gain  :Change microphone gain per connection <gain
-                              [gain [...]]>
+     discover                  :Discover CAS
+     cancel                    :CAP commander cancel current procedure
+     change_volume             :Change volume on all connections <volume>
+     change_volume_mute        :Change volume mute state on all connections <mute>
+     change_volume_offset      :Change volume offset per connection <volume_offset
+                                [volume_offset [...]]>
+     change_microphone_mute    :Change microphone mute state on all connections <mute>
+     change_microphone_gain    :Change microphone gain per connection <gain
+                                [gain [...]]>
+     broadcast_reception_start : Start broadcast reception on one connection with <address>
+                                 and <type>, using <adv_sid> and <broadcast_id>, optionally
+                                 <pa_interval>, <sync_bis> and <metadata>
```

Comment:
Arguably outside the scope of this PR, but this should really have been `"SOME DESCRIPTION" "ARGUMENTS"` as is commonly the case for CLIs.